README.md

```diff
@@ -56,9 +56,35 @@ then, we can easily validate the above API using `Zerocode` like below.
 
 + Using YAML described as below,
 
-> _The beauty here is, we can use the payload/headers structure as it is without any manipulation._
+> _The beauty here is, we can use the payload/headers structure as it is without any manipulation or a JSON path pointing to the field._
+
+## Validators
 
 ```yaml
+
+---
+url: api/v1/customers/123
+method: GET
+request:
+  headers:
+    Content-Type: application/json
+retry:
+  max: 3
+  delay: 1000
+validators:
+- field: "$.status"
+ value: 200
+- field: "$.body.type"
+ value: Premium Visa
+- field: "$.body.addresses[0].line1"
+ value: 10 Random St
```

Comment:
`value` field should be indented with one more space.
Can you please fix it?

Reply:
fixed now